RELEASE.md

```diff
@@ -1,3 +1,13 @@
+# Release 2.11.1
+
+A patch to replace the deprecated OOB auth flow, and remove deprecated aliases
+that were removed in numpy 1.24.0.
+
+## Features
```

Comment:
I'd phrase these as bug fixes rather than features - patch releases typically should be limited to bug fixes, and I think it's reasonable to characterize both of these as bug fixes or avoiding regressions.

E.g. something like

```
## Bug Fixes

- Prevent regression in TensorBoard.dev uploader authentication by replacing deprecating OOB auth flow with limited-input device flow (... links, etc.)
- Fix compatibility with numpy 1.24.0 by removing deprecated type aliases (#6117, #6140)
```